core/src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java

```diff
@@ -239,7 +247,7 @@ public static TimeOperator2 from(TimeOperator to) {
         Solid;
 
         public static String asString(GeometryOperand go) {
-            return "gml:" + go.name();
+            return Joiner.on(":").join(GmlConstants.NS_GML_PREFIX, go.name());
```

Comment:
I think that may be a little bit too much overhead for concatenating **two** strings....